docs/writing-tests/testharness.md

```diff
@@ -25,6 +25,10 @@ documented in two sections:
 See [server features](server-features) for advanced testing features that are commonly used
 with testharness.js. See also the [general guidelines](general-guidelines) for all test types.
 
+This page describes testharness.js exhaustively; [the tutorial on writing a
+testharness.js test](testharness-tutorial) offers a more limited but grounded
+guide to the process.
```

Comment:
Seems like a slightly odd way of phrasing this. When describing what the tutorial does, it is good to think of the user need it fulfills first. It's basically a concise tutorial for those who are just getting started and don't want all the things just yet, right? So maybe something like "[the tutorial on writing a
testharness.js test](testharness-tutorial) provides a concise guide to writing a test — a good place to start for newcomers to the project."?

Reply:
Yes, great!